spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -9,9 +9,12 @@ such as creating and running stream applications.
 
 == Introducing Spring Cloud Stream
 
-Spring Cloud Stream allows developing and running messaging microservices easily using Spring Integration, by just adding `@EnableBinding` and running your app as a Spring Boot application. Spring Cloud Stream applications connect to the physical broker through bindings, which link Spring Integration channels to physical broker destinations, or either input (consumer bindings) or output (producer bindings). The creation of the bindings, and therefore their broker-specific implementation is handled by a binder, which isanother important abstraction of Spring Cloud Stream. Binders abstract out the broker-specific implementation details. In order to connect to a specific type of broker (e.g. Rabbit or Kafka) you just need to have the relevant binder implementation on the classpath.
+Spring Cloud Stream is a framework for building message-driven microservices, and more specifically stream processing applications.
```

Comment:
let's make it a small one liner by removing `and more specifically stream processing applications.`  and move mention of specific stream functionality to after the next sentence...